tensorflow/python/keras/callbacks_test.py

```diff
@@ -849,6 +850,24 @@ def test_LearningRateScheduler(self):
           float(keras.backend.get_value(
               model.optimizer.lr)) - 0.01 / 4) < keras.backend.epsilon()
 
+      cbks = [keras.callbacks.LearningRateScheduler(lambda epoch, _: cosine_decay(
+          0.01, epoch, 2))]
+      model.compile(
+          loss='categorical_crossentropy',
+          optimizer='sgd',
+          metrics=['accuracy'])
+      model.fit(
+          x_train,
+          y_train,
+          batch_size=BATCH_SIZE,
+          validation_data=(x_test, y_test),
+          callbacks=cbks,
+          epochs=2,
+          verbose=0)
+      assert (
```

Comment:
since you're doing cosine decay, can this be a test that actually decayed the learning rate? seems it stays at 0.01?

Reply:
changed